ci/build.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.19.2
+    tag: 1.20.5
```

Comment:
You might have issues with 1.20.5 - see how it goes.

Reply:
I left this as it is to see how it goes.